### ISSUE TITLE
about testing

### ISSUE THREAD

**Issue by Xchangjiang** [VisionLearningGroup/R-C3D#45]
I have already downed the "activitynet_iter_135000.caffemodel", but when I prepare to generate the prediction log file on the validation set, and run "./experiments/activitynet/test/script_test.sh" , some questions occured,it said "Waiting for ./experiments/activitynet/snapshot/activitynet_iter_135000.caffemodel to exist...", and trapped in a dead circle.  I can't understand this question, could you please explain it?
Thanks for your kindness!

**Comment by huijuan88** [comment]
Did you set the right path?


On Nov 30, 2018, at 20:13, WangChangjiang <notifications@github.com<mailto:notifications@github.com>> wrote:


I have already downed the "activitynet_iter_135000.caffemodel", but when I prepare to generate the prediction log file on the validation set, and run "./experiments/activitynet/test/script_test.sh" , some questions occured,it said "Waiting for ./experiments/activitynet/snapshot/activitynet_iter_135000.caffemodel to exist...", and trapped in a dead circle. I can't understand this question, could you please explain it?
Thanks for your kindness!

**Comment by Xchangjiang** [comment]
yes, I put  the caffemodel under the derectory, which is "mypath/R-C3D-master/snapshot/ "

**Comment by Xchangjiang** [comment]
I have sloved this question. thank you. but now I get a new error: F1201 17:15:49.549661  9384 syncedmem.hpp:18] Check failed: error == cudaSuccess (30 vs. 0)  unknown error
I can't solve it..Do you know why?  

my nvidia driver is 390.87, cuda 8.0 and cudnn 5.1.10

**Comment by Xchangjiang** [comment]
I also have sloved this question. thank you. I am building for Python 3 and running from Python 3 , I want to know can I run this project properly?

**Comment by HanwGeek** [comment]
> I also have sloved this question. thank you. I am building for Python 3 and running from Python 3 , I want to know can I run this project properly?

你好，我想请问下如何解决"error == cudaSuccess"问题？ 非常感谢！
